### ISSUE TITLE
Please add site https://www.38xs.com/

### ISSUE THREAD

**Issue by Finks82** [dteviot/WebToEpub#1081]
pls add https://www.38xs.com/

**Comment by dteviot** [comment]
Annoying.  It's almost, but not quite, identical to https://www.230book.net/book/5483/.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://www.38xs.com/200692/, chapters 1 &amp; 2

For my notes: 37 minutes work

**Comment by Finks82** [comment]
on some only half chapter is download example: https://www.38xs.com/58639/54104100.html , its one of those site that use 2 page for one same chapter

**Comment by dteviot** [comment]
@Finks82 

I think I've fixed it.  Note, I can't read Chinese, so it's a hard for me to be sure the epub has the combined chapter.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://www.38xs.com/58639/, chapters 6 &amp; 7

For my notes: 75 minutes work (total)
